orchagent/portsorch.cpp

```diff
@@ -817,10 +871,7 @@ void PortsOrch::doVlanMemberTask(Consumer &consumer)
         {
             if (vlan.m_members.find(port_alias) != vlan.m_members.end())
             {
-                /* Assert the port belongs the a VLAN */
-                assert(port.m_vlan_id && port.m_vlan_member_id);
-
-                if (removeVlanMember(vlan, port))
+                if (removeVlanMember(vlan, port) && removeBridgePort(port))
```

Comment:
if there is no vlan member, then we can removeBridgePort.

Reply:
done.